app/src/split/bluetooth/central.c

```diff
@@ -263,14 +302,34 @@ static uint8_t split_central_chrc_discovery_func(struct bt_conn *conn,
         slot->subscribe_params.value_handle = bt_gatt_attr_value_handle(attr);
         slot->subscribe_params.notify = split_central_notify_func;
         slot->subscribe_params.value = BT_GATT_CCC_NOTIFY;
-        split_central_subscribe(conn);
+        split_central_subscribe(conn, &slot->subscribe_params);
+#if ZMK_KEYMAP_HAS_SENSORS
+    } else if (bt_uuid_cmp(((struct bt_gatt_chrc *)attr->user_data)->uuid,
```

Comment:
Since this is used repeatedly here, it would improve readability a bit to do something like

```c
const struct bt_uuid* chrc_uuid = ((const struct bt_gatt_chrc*)attr->user_data)->uuid;

if (bt_uuid_cmp(chrc_uuid, ...) == 0) {
  ...
} else if (bt_uuid_cmp(chrc_uuid, ...) == 0) {
  ...
} else if ...
```

Reply:
Much cleaner. Moved to that.